crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -18,85 +18,88 @@ pub(super) fn place_comment<'a>(
     comment: DecoratedComment<'a>,
     locator: &Locator,
 ) -> CommentPlacement<'a> {
-    // Handle comments before and after bodies such as the different branches of an if statement.
-    let comment = if comment.line_position().is_own_line() {
```

Comment:
What did you change here? It's hard to understand the changes because of the added indent

Reply:
`handle_parenthesized_comment` is the first branch, followed by an `or_else` with these two cases, followed by the existing `or_else` for the enclosed node.